pyproject.toml

```diff
@@ -34,7 +34,8 @@ ipywidgets = "8.1.2"
 optional = true
 
 [tool.poetry.group.docs.dependencies]
-pdoc3 = "0.10.0"
+sphinx = "7.4.7"
```

Comment:
Should we use newer sphinx?

Reply:
`7.4.7` is the latest version of Sphinx that is still supported in Python 3.9